spec/book_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  subject(:book) { create(:book) }
+
+  describe '#create' do
+    context 'When the book is properly set' do
+      it do
+        is_expected.to be_valid
+      end
+    end
+
+    context 'When the book is created' do
+      it do
+        %i[author title genre image publisher year].each do |attr|
+          is_expected.to validate_presence_of(attr)
```

Comment:
Ok, you use this gem is right we use this usually, but here the goal of the exercise was to do test on other to understand better how it's work.
No matter, we will use this now.
However by convention for this kind of test we do not use loop. We write a line for each validation. So, if I may, I will ask you to write a line for each validation.